specification/attributes/numeric_format.md

```diff
@@ -0,0 +1,91 @@
+# Numeric Format
+
+Columns that provide numeric values conforming to specified rules and formatting requirements ensure clarity, accuracy, and ease of interpretation for both humans and systems.
+
+All columns capturing a numeric value, defined in the FOCUS specification, MUST be a real number represented as an integer or a decimal value as defined in this section.
+
+## Attribute ID
+
+NumericFormat
+
+## Attribute name
+
+Numeric Format
+
+## Description
+
+Rules and formatting requirements for numeric columns appearing in billing data.
+
+## Requirements
+
+* Columns with a Numeric format MUST contain a single, numeric value.
+* Numeric values MUST be expressed as integer or decimal values. Fractional notation is NOT allowed.
+* Numerical values expressed with mathematical symbols, operators, or exponent values are not permitted.
+* Numeric values MUST NOT contain qualifiers or additional characters (e.g. currency symbols, units of measure, etc.).
+* Numeric values MUST NOT contain commas or punctuation marks except for a single decimal point if required to express a decimal value.
+* Numeric values MUST NOT include a character to represent sign for a positive value.  The only sign permitted is a negative sign (-) to indicate a negative value.
```

Comment:
Not a 100% sure if one of the BCP-14 values should be used for the second sentence. 'MAY' is optional but in the case of negative values, that's not optional So would we say "Negative sign (-) MUST be used to indicate a negative value."?